### DISCUSSION TITLE
Any plans on a Nova Tool to manage these settings?

### DISCUSSION THREAD

**Discussion by ragingdave** [spatie/laravel-settings#38] [Q&A]
Basically just a started question, or maybe spark for someone to potentially build something cool for Nova level management.

**Comment by rubenvanassche** [comment]
Hi @ragingdave,

Not at the moment, though I think it's really cool to have such thing. It also is a complicated thing to build and requires I think a lot of time. If you would make such a package, I would happily add a link in the readme to it.

**Reply by ragingdave** [reply]
I actually have a working POC, that I'm developing for my company's usage. I'd have to see if I'd be able to open source it before I get it out there. That being said, it would definitely be nice to have some community help to get all the bells and whistles.

**Reply by baurelien** [reply]
Hi @ragingdave ,

Do you still plan to open source your work on this Nova tool ?

**Comment by ragingdave** [comment]
I will get on getting the ok to open source it and hopefully will.

**Reply by dmason30** [reply]
@ragingdave Did you end up having to specify the Nova fields for each setting or did you rely on the settings type hint. I think we want to switch to this package but also need for away to manage them in Nova.
